Modelica/Fluid/Sources.mo

```diff
@@ -29,7 +29,7 @@ package Sources "Define fixed or prescribed boundary conditions"
       annotation (Dialog(group="Boundary temperature or boundary specific enthalpy",
                   enable = not use_T));
     parameter Medium.MassFraction X[Medium.nX](
-         quantity=Medium.substanceNames) = Medium.X_default
+         quantity="MassFraction." + Medium.substanceNames) = Medium.X_default
```

Comment:
Trying to concretize @HansOlsson's alternative (but GitHub won't let me include the line about in the suggestion, making it look silly):
```suggestion
         ) = Medium.X_default
```

Reply:
Applying this change to keep this PR aligned with #4100.